api/v01/entities/vrp_result.rb

```diff
@@ -56,71 +53,83 @@ class VRPResultDetailedCosts < Grape::Entity
 
     class VrpResultSolutionRouteActivities < Grape::Entity
       expose :point_id, documentation: { type: String, desc: 'Linked spatial point' }
-      expose :travel_distance, documentation: { type: Integer, desc: 'travel distance from the previous point' }
-      expose :travel_duration, documentation: { type: Integer, desc: 'travel time from the previous point' }
-      expose :waiting_duration, documentation: { type: Integer, desc: '' }
-      expose :arrival_time, documentation: { type: Integer, desc: '' }
+      expose :travel_distance, documentation: { type: Integer, desc: 'Travel distance from previous point' }
+      expose :travel_time, documentation: { type: Integer, desc: 'Travel time from previous point' }
+      expose :travel_value, documentation: { type: Integer, desc: 'Travel value from previous point' }
+      expose :waiting_time, documentation: { type: Integer, desc: '' }
+      expose :begin_time, documentation: { type: Integer, desc: 'Time visit starts' }
+      expose :end_time, documentation: { type: Integer, desc: 'Time visit ends' }
       expose :departure_time, documentation: { type: Integer, desc: '' }
-      expose :service_id, documentation: { type: String, desc: '' }
-      expose :pickup_shipment_id, documentation: { type: String, desc: '' }
-      expose :delivery_shipment_id, documentation: { type: String, desc: '' }
+      expose :service_id, documentation: { type: String, desc: 'Internal reference of the service' }
+      expose :original_service_id, documentation: { type: String, desc: '' }
+      expose :pickup_shipment_id, documentation: { type: String, desc: 'Internal reference of the shipment' }
+      expose :delivery_shipment_id, documentation: { type: String, desc: 'Internal reference of the shipment' }
+      expose :original_shipment_id, documentation: { type: String, desc: '' }
       expose :detail, using: VrpResultSolutionRouteActivityDetails, documentation: { desc: '' }
+      expose :type, documentation: { type: String, desc: 'Start, end, break, service, pickup or delivery' }
```

Comment:
Start -> start

Have a 'depot' type instead of 'start' or 'end' could allow to have multiple trips within the same route ?

Reply:
break should be rest to be consistent with the API and the internal model ?

Reply:
Today API returns 'break', 'start' and 'end' (ortools.rb) that is why I just adapted documentation accordingly. But you are right, we should change these attributes names in version 2.

Reply:
I see no previous mention of the field type in the solution returned by ortools.rb before this commit ?
These elements are not part of the solution returned by the API, they are specific to the model used with OR-Tools.
For OR-Tools a route always have a start and a end regardless the vehicle has depots or not.
break is the terminology used by OR-Tools. the parse_output method return a rest_id.

Reply:
I am sorry, you are right ! I got confused because I know I documented _type_ because we already return a type for some elements of _unassigned_ but I did not remember that we never return 'break', 'start' nor 'end'.

Therefore you are right, we could change 'break', 'start' and 'end' with 'rest' and 'store'. To be consistent. But we won't be consistent with ORtools returned values anymore. But I guess the most important is to be consistent for users.

Reply:
Yes, the API doesn't have to be consistent with OR-Tools, we wrap our own model to the format expected by OR-Tools and convert back the answer to our format.